webete/webete.py

```diff
@@ -93,6 +109,10 @@ def main():
     action_group = parser.add_mutually_exclusive_group(required=True)
     action_group.add_argument('-a', '--auto', action='store_true', help='Automatic simple recon')
     action_group.add_argument('-p', '--python', dest='python', metavar='FILE', help='Look for a specific pyc file under __pycache__ subdir')
+    auth_group = parser.add_argument_group(title="Authentication", description="Autentication and session handling options")
```

Comment:
Need to add a `--http-basic-something-bla` argument to `store_true` that the user wants basic auth. If that's the case, then we can look into user/pw.
This way we can also decouple the user/pw from the actualy type of auth without having to put stuff like `--digest-auth-pass` bla bla.